examples/glue_benchmark/glue.py

```diff
@@ -23,86 +27,39 @@
 
 import keras_nlp
 
-FLAGS = flags.FLAGS
+seed = 42
+os.environ["PYTHONHASHSEED"] = str(seed)
+tf.random.set_seed(seed)
+
 
 flags.DEFINE_string(
-    "task_name",
+    "task",
     "mrpc",
     "The name of the GLUE task to finetune on.",
 )
-
-flags.DEFINE_integer(
-    "batch_size",
-    32,
-    "Batch size of data.",
-)
-
-flags.DEFINE_integer(
-    "epochs",
-    2,
-    "Number of epochs to run finetuning.",
-)
-
-flags.DEFINE_float(
-    "learning_rate",
-    5e-5,
-    "Learning rate",
-)
-
+flags.DEFINE_string("model", None, "The Model you want to train and evaluate.")
```

Comment:
You can use the same flag description as in benchmarks/sentiment_analysis.py: "The name of the classifier such as BertClassifier."